functions/go/set-namespace-v2/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-namespace-v2
+
+go 1.16
+
+require github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220315215931-2ef5e56080fb // indirect
```

Comment:
needs to be updated once https://github.com/GoogleContainerTools/kpt-functions-sdk/pull/531 is merged.